initrd.txt

```diff
@@ -0,0 +1,1397 @@
+[   12539904]  ./rootfs-0/kernel/x86/microcode/GenuineIntel.bin
+[    5343472]  ./rootfs-1/usr/lib64/libcrypto.so.3*
+[    3208664]  ./rootfs-1/usr/lib/systemd/libsystemd-shared-252.so*
+[    2591624]  ./rootfs-1/usr/lib/gcc/x86_64-cros-linux-gnu/13/libstdc++.so.6.0.32*
+[    2058480]  ./rootfs-1/usr/lib/systemd/libsystemd-core-252.so*
+[    1933576]  ./rootfs-1/usr/lib64/libc.so.6*
+[    1845305]  ./rootfs-1/usr/lib/firmware/phanfw.bin
+[    1649076]  ./rootfs-1/usr/lib/firmware/mellanox/mlxsw_spectrum-13.2010.1006.mfa2
+[    1649016]  ./rootfs-1/usr/lib/systemd/systemd-networkd*
+[    1359116]  ./rootfs-1/usr/lib/firmware/mellanox/mlxsw_spectrum2-29.2010.1006.mfa2
+[    1167440]  ./rootfs-1/usr/sbin/nvme*
+[    1145040]  ./rootfs-1/usr/lib64/libgcrypt.so.20.4.3*
+[    1127648]  ./rootfs-1/usr/lib/firmware/mellanox/mlxsw_spectrum3-30.2010.1006.mfa2
+[    1038064]  ./rootfs-1/usr/sbin/btrfs*
```

Comment:
That binary could be symlinked to `/sysusr/`, same for xfs* and mkfs.* stuff and even `ip` and `mdadm`